.buildkite/pipeline.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3
+# Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
If we also wanted to do the same for the performance pipelines, we would need to create another pipeline file?
Maybe would be worth to rename this to ci-pipeline.py?

Reply:
Yeah, not sure how to best do that at the moment... We could have different arguments.. `pipeline.py --pr`, `pipeline.py --perf`, or as you say a different script, using some common functionality.

Reply:
PyLint complains about about `pipeline-pr.py` .. will do `pipeline_pr.py` to avoid the error.